posts/2024-03-11-crates-io-download-changes.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "crates.io: Download changes"
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+[crates.io](https://crates.io) is the official package registry of the Rust community where developers can publish their Rust packages for others to use. It plays a crucial role in the Rust ecosystem by providing a centralized platform for package distribution.
+
+The Rust community has been growing rapidly, and with it, the number of downloads from crates.io has been increasing by 200% to 300% per year.
```

Comment:
This might be a level of table setting we don't need on the Rust blog? (Yes, I'm aware of the irony of this coming from _me_, the king of the useless, wordy aside.)

I think we could frame this and tease the solution more concisely:

```suggestion
Like the rest of the Rust community, [crates.io](https://crates.io) has been growing rapidly, with download and package counts increasing 2-3x year on year. This growth doesn't come without problems, and we have made some changes to download handling on crates.io to ensure we can keep providing crates for a long time to come.
```

Reply:
3-4x, right? A 100% increase is a 2x the old value.

Reply:
@bjorn3 that's a good point 😅 

it is 2x-3x (+100% to +200%). let me fix that...